yt/visualization/volume_rendering/scene.py

```diff
@@ -399,6 +401,9 @@ def save_annotated(
         label_fmt : str, optional
            A format specifier (e.g., label_fmt="%.2g") to use in formatting
            the data values that label the transfer function colorbar.
+        label_fontsize : int, optional
```

Comment:
the docstring should mention what the default value is.

Reply:
updated

Reply:
... and now it should be synced with the type annotation.
It'd be worth linking to https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.tick_params.html#matplotlib.axes.Axes.tick_params
too, so users get first hand documentation for what strings mean in this context.
But maybe we could just type this as `float` and hide this implementation detail.

(sorry that it's taking so many iterations, I promise we'll converge after this one)

Reply:
I know a couple docstring where urls to matplotlib doc would be very useful, but here it's not crucial. Thanks !